ArduCopter/events.cpp

```diff
@@ -286,6 +286,36 @@ void Copter::failsafe_terrain_on_event()
     }
 }
 
+// check for tether fault
+void Copter::failsafe_tether_status_check()
+{
+    bool tether_trigger = copter.planck_interface.get_spool_status() == PLANCK_DECK_SPOOL_RETRIEVE;
+
+    if (tether_trigger != failsafe.tether) {
+        if (tether_trigger) {
+            failsafe_tether_on_event();
+        } else {
+            AP::logger().Write_Error(LogErrorSubsystem::FAILSAFE_TETHER, LogErrorCode::ERROR_RESOLVED);
+            failsafe.terrain = false;
```

Comment:
?

Reply:
should be failsafe.tether = false:, will change